pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -386,7 +402,29 @@ protected ConsumerImpl(PulsarClientImpl client, String topic, ConsumerConfigurat
 
         topicNameWithoutPartition = topicName.getPartitionedTopicName();
 
+        InstrumentProvider ip = client.instrumentProvider();
+        consumersOpenedCounter = ip.newCounter("pulsar.client.consumer.opened", Unit.Sessions,
+                "Counter of sessions opened", topic, Attributes.empty());
```

Comment:
Since we provided OTEL metrics, can we provide it more detail?
I mean, the metrics at the TOPIC level are a bit coarse-grained

Reply:
I'm not sure I understand the comment. The counter here is getting tagged with tenant/namespace/topic/partition attributes. By default, we only emit `tenant` and `namespace` and user would be able to select the desired level

Reply:
I mean for consumer metrics, should we expose them in subscription level?

Reply:
Good point. I've added `pulsar.subscription` attribute